docs/pattern-pseudo-patterns.md

```diff
@@ -59,3 +59,7 @@ Now when we generate our site we'll have our homepage template rendered twice. O
 By default, pseudo-patterns **cannot** be used as pattern includes. The data included in the pseudo-pattern, the bit that actually controls the magic, cannot be accessed when rendering the pattern include.
 
 To utilize pseudo-patterns as pattern includes for the PHP version of Pattern Lab you can install the [Data Inheritance Plugin](https://github.com/pattern-lab/plugin-php-data-inheritance).
+
+## Re-ordering Pseudo-Patterns
+
+To learn how to re-order pseudo-patterns, check [Reorganizing Patterns](pattern-reorganizing.md).
```

Comment:
I'm not sure if that link will resolve to the appropriate URL. I'd recommend giving it an absolute URL of `/docs/pattern-reorganizing.html`